server/src/resolvers/answer.js

```diff
@@ -123,45 +111,34 @@ module.exports = {
       )
     },
     updateAnswerAndSources: async (_, { id, content, sources }, ctx, info) => {
-      // Imports the Google Cloud client library
+
       const { Translate } = require('@google-cloud/translate');
 
-      // Your Google Cloud Platform project ID
       const projectId = 'YOUR_PROJECT_ID';
 
-      // Instantiates a client
       const translate = new Translate({
         projectId: projectId,
       });
 
-      // Creation contentTab
       const contentTab = [];
 
-      // The text to translate
       const text = content;
 
-      // The target language
       const targeten = 'en';
       const targetfr = 'fr';
 
-      // Translates some text into English
       await translate
         .translate(text, targeten)
         .then(results => {
           const translationen = results[0];
           contentTab.push({ text: translationen, lang: targeten });
-
-          console.log(`Text: ${text}`);
-          console.log(`Translation en: ${translationen}`);
         })
```

Comment:
If you use `await` you dont have to use `then`, you can write it like that:
```javascript
const results = await translate.translate(text, targeten)
const translationen = results[0];
contentTab.push({ text: translationen, lang: targeten });
```

Reply:
It's noted, thanks :)